File: samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/TokenExchange/TokenExchangeConfig.cs
﻿// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace VirtualAssistantSample.TokenExchange
{
    public class TokenExchangeConfig : ITokenExchangeConfig
    {
        public string Provider { get; set; }

        public string ConnectionName { get; set; }
    }
}

File: templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/src/tokenExchange/tokenExchangeConfig.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

export interface ITokenExchangeConfig {
    provider: string;
    connectionName: string;
}

export class TokenExchangeConfig implements ITokenExchangeConfig {
    public provider = '';
    public connectionName = '';
}
